packages/utils/src/atoms/partial-date-time.ts

```diff
@@ -17,9 +17,7 @@ export const formatDateExplicitMonth = (date: Date): string =>
 export const formatTime = (date: Date): string =>
   date.toLocaleString(locale, { hour: hourFormat, minute: minuteFormat, hour12: false }).split(' ').join(':')
 
```

Comment:
This somehow made it to main, wasn't letting me push up my branch without chagning... just a reformat